### PR TITLE
Chart branch

### PR DESCRIPTION
**Pull request by nitinramadoss**
Adds pom dependencies
Adds chart of hobbies to page

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -96,25 +96,33 @@ <h2>Goals </h2>
...
              <br>
              <button type="button" onclick="removePhrase()">Clear Comments</button>
              </form>
-        <ul id = "section"></ul>
+        <ul id="section"></ul>
        </div>
     </div>
+
+     <script src="https://www.gstatic.com/charts/loader.js"></script>
+        <script>
```
Comment (jasperlee-google) [inline portfolio/src/main/webapp/index.html:120]: Is there a reason why this `<script>` tag is indented?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -96,25 +96,33 @@ <h2>Goals </h2>
...
+        google.charts.load('current', {packages: ['corechart']});
+        google.charts.setOnLoadCallback(loadChart);
+        </script>
```
Comment (lee12jin) [inline portfolio/src/main/webapp/index.html:123]: Same comment as jaspers above.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -37,5 +35,21 @@ async function removePhrase(){
...
+      stats.addColumn('number', 'Percentage');
+      stats.addRows([
+        ['Programming', 0.65],
+        ['Biking', 0.10],
+        ['Gaming', 0.25]
+      ]);
+
+      // Instantiate and draw the chart.
+      var chart = new google.visualization.PieChart(document.getElementById('activityPieChart'));
+      chart.draw(stats, null);
```
Comment (lee12jin) [inline portfolio/src/main/webapp/script.js:51]: Indentation looks off

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (nitinramadoss) [reply portfolio/src/main/webapp/index.html:120]: I'll fix the indentation for the next PR, thank you for approving :)

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (nitinramadoss) [reply portfolio/src/main/webapp/script.js:51]: I'll fix it, thank you for approving :)

**Pull request merged**